static/app/views/settings/organizationDeveloperSettings/sentryFunctionRow/sentryFunctionRowButtons.tsx

```diff
@@ -0,0 +1,15 @@
+import {Organization, SentryFunction} from 'sentry/types';
+
+import ActionButtons from './actionButtons';
+
+type Props = {
+  onClickRemove: (org: Organization, sentryFn: SentryFunction) => void;
+  org: Organization;
+  sentryFn: SentryFunction;
+};
+
+const SentryFunctionRowButtons = ({org, sentryFn, onClickRemove}: Props) => {
+  return <ActionButtons org={org} sentryFn={sentryFn} onDelete={onClickRemove} />;
+};
```

Comment:
Do we really need this wrapper? looks like you could just use ActionButtons in place of this